app/models/api_namespace.rb

```diff
@@ -226,12 +229,12 @@ def self.import_as_json(json_str)
           api_form_hash = hash['api_form'].except(*neglected_attributes[:api_form]).merge({'api_namespace_id': new_api_namespace.id})
           ApiForm.create!(api_form_hash)
         end
-        
-        # Creating api_clients
-        if hash['api_clients'].present? && hash['api_clients'].is_a?(Array)
-          hash['api_clients'].each do |api_client_hash|
-            api_client_hash = api_client_hash.except(*neglected_attributes[:api_client]).merge({'api_namespace_id': new_api_namespace.id})
-            ApiClient.create!(api_client_hash)
+
+        # Creating api_namespace_keys
```

Comment:
Here as well, while importing from JSON, I think ApiKeys also needs to be imported.